community/modules/scheduler/htcondor-central-manager/versions.tf

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 3.83"
+    }
+    time = {
+      source  = "hashicorp/time"
+      version = "~> 0.9"
+    }
+  }
+  provider_meta "google" {
+    module_name = "blueprints/terraform/hpc-toolkit:htcondor-central-manager/v1.20.0"
+  }
+
+  required_version = ">= 0.13.0"
```

Comment:
nit: I thought we decided on 1.0 as the minimum everywhere. I know we still have stragglers but for new modules I think 1.0 is more appropriate.

Reply:
I've updated to 1.1 (which is accurate). My general feeling is that the child modules ought to be accurate, although 1.0 has been out 2 years now so it's not unrealistic to impose that full-stop.